c7n/resources/ecs.py

```diff
@@ -483,8 +483,10 @@ def get_resources(self, ids, cache=True):
     def augment(self, resources):
         results = []
         client = local_session(self.session_factory).client('ecs')
+        retry = get_retry(('ThrottlingException',))
         for task_def_set in resources:
-            response = client.describe_task_definition(
+            response = retry(
```

Comment:
any reason for not using self.manager.retry  instead of constructing a new retry?

Reply:
Good question. I noticed that a manager is available in that context. I'll refactor to use `QueryResourceManager.retry`.

Reply:
Resolved in 8f51e51